luatest/replica_proxy.lua

```diff
@@ -0,0 +1,137 @@
+--- Manage connection to tarantool replication instances via proxy.
+--
+-- @module luatest.replica_proxy
+
+local checks = require('checks')
+local fiber = require('fiber')
+local log = require('log')
+local socket = require('socket')
+
+local Connection = require('luatest.replica_conn')
+
+local TIMEOUT = 0.001
+local BACKLOG = 512
+
+local Proxy = {
+    constructor_checks = {
+        client_socket_path = 'string',
+        server_socket_path = 'string',
+        process_client = '?table',
+        process_server = '?table',
+    },
+}
+
+function Proxy:inherit(object)
+    setmetatable(object, self)
+    self.__index = self
+    return object
+end
+
+--- Build proxy object.
+--
+-- @param object
+-- @string object.client_socket_path Path to client Unix socket.
+-- @string object.server_socket_path Path to server Unix socket.
+-- @tab[opt] object.process_client TBD.
```

Comment:
process_client and process_server description can be found in https://github.com/tarantool/tarantool/commit/b907e7260920b05886630d1a4e057d865367b403 commit message.

Reply:
Thanks for pointing it out.